src/js/components/question-input/question-input.js

```diff
@@ -0,0 +1,175 @@
+import Preview from "./preview";
+import "./question-input.scss";
+
+import { SimpleSelect } from "react-selectize";
+import { buildOptions } from "../../../lib/helpers/questions-helpers";
+
+export const TEXTAREA_PLACEHOLDER = "Enter your line here.";
+export const TYPE_OPTIONS = buildOptions(['Code', 'Text']);
+export const VIEW_OPTIONS = buildOptions(['Normal', 'Italics', 'Bolded']);
+
+export default class QuestionInput extends React.Component {
+	constructor(props) {
+	    super(props);
+	    this.typeOptions = TYPE_OPTIONS;
```

Comment:
no need to initialize these in the constructor or anything. in the component, they can just be accessed by regular var name :D